app/src/sharedTest/java/org/oppia/app/help/HelpFragmentTest.kt

```diff
@@ -38,6 +40,30 @@ class HelpFragmentTest {
     FirebaseApp.initializeApp(ApplicationProvider.getApplicationContext())
   }
 
+  private fun createHelpActivityIntent(profileId: Int, isFromExploration: Boolean): Intent {
+    return HelpActivity.createHelpActivityIntent(
+      ApplicationProvider.getApplicationContext(),
+      profileId,
+      isFromExploration
+    )
+  }
+
+  @Test
+  fun testHelpFragment_getIsFromExplorationTrue_checkBackArrowVisible() {
+    launch<HelpActivity>(createHelpActivityIntent(0, true)).use {
+      onView(withContentDescription(R.string.abc_action_bar_up_description))
+        .check(matches(isCompletelyDisplayed()))
+    }
+  }
+
+  @Test
+  fun testHelpFragment_getIsFromExplorationFalse_checkBackArrowNotVisible() {
```

Comment:
Rename to `testHelpFragment_parentIsNotExploration_checkBackArrowNotVisible `

Reply:
Done